.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
```

Comment:
This is the configuration to run flake8 and pylint in Travis CI (https://travis-ci.com/WHRB/whrbcatalog)